src/dftbp/dftb/hybridxc.F90

```diff
@@ -84,14 +85,17 @@ module dftbp_dftb_hybridxc
   !> Enumerator for algorithms to build up the Fock-type exchange contribution to the Hamiltonian.
   type :: THybridXcAlgoEnum
 
+    !> None
+    integer :: none = 0
```

Comment:
As this is now defined,  why not set the default in THybridXcFunc to none? Likewise for the THybridXcGammaTypesEnum (add none and use as default)

Reply:
done

Reply:
@bhourahine I had to revert that change, since NAG fails with an internal compiler error in this case.
@aradi Will try to break things down to a minimal example and file a bug report.

Reply:
@bhourahine I restored the version that sets the default values, as the NAG team has already fixed the bug.